src/nodes/ImageView.vue

```diff
@@ -224,16 +226,13 @@ export default {
 
 			return this.loaded && this.imageLoaded
 		},
-		imageFileId() {
-			return getQueryVariable(this.src, 'fileId')
-		},
 		isSupportedImage() {
-			return typeof this.mime === 'undefined'
-				|| IMAGE_MIMES.indexOf(this.mime) !== -1
+			return typeof this.mimeType === 'undefined'
+				|| [...IMAGE_MIMES, ...MEDIA_MIMES].indexOf(this.mimeType) !== -1
```

Comment:
I don't think this is necessary. Just leave it as is.

Reply:
Will fix it.